core-processor/src/executor.rs

```diff
@@ -203,9 +203,10 @@ fn get_pages_to_be_updated<A: ProcessorExt>(
 
 /// Execute wasm with dispatch and return dispatch result.
 pub fn execute_wasm<A: ProcessorExt + EnvExt + IntoExtInfo + 'static, E: Environment<A>>(
```

Comment:
One thing forgot to notice:
this:
https://github.com/gear-tech/gear/blob/973250632d8b4163c85c6c10dc264b0c43f044cd/core-processor/src/executor.rs#L278-L294
should also be simplified